contents/newsletter/when-product-velocity-breaks-speed-of-adoption.md

```diff
@@ -0,0 +1,217 @@
+
+- Adding so much to the UI that it subtracts from [user experience](/blog/redesigned-nav-menu)
+
+**Instead:**
+
+- Define [activation criteria](/product-engineers/activation-metrics) for new products, and track user behavior over time
+
+- Leverage AI to surface features when a user is already doing the related job
+
+- Embed docs within your product to keep users in flow (linking to an external help center is a form of context switching)
+
+- Leverage habit loops (trigger → action → reward → reinforcement) and other [PLG](https://productled.com/blog/product-led-growth-framework-for-saas-companies) tactics to keep users engaged
+
+- Commit to [continuous discovery](https://www.youtube.com/watch?v=9RFaz9ZBXpk). Let user feedback and behavior influence what gets amplified next
+
+### 4. Measure learning, not just usage
+
+Adoption doesn’t really start with features. It starts with people getting better at their jobs. That’s why a lot of our marketing content isn’t really about PostHog at all – it’s about [how to be a better product engineer](https://newsletter.posthog.com/). 
+
+**What failure looks like:**
+
+- Overemphasizing vanity metrics like clicks, impressions and daily active users (DAU)
+
+- Incentivizing your product team to be a [feature factory](https://xenoss.io/blog/product-velocity-trap-solutions) instead of taste makers
+
+- Publishing content that only talks about your product – instead of the market, community, and problems your users actually think about
+
+- Believing other people care about your product as much as you do (they don’t)
+
+**Instead:**
+
+- Publish [learnings](/blog/8-learnings-from-1-year-of-agents-posthog-ai), [works-in-progress](/blog/multi-modal-llm-user-behavior-analysis), and durable mental models that are useful with, or without your product 
+
+- Teach others about your domain (Hubpsot teaches you how to be a [better marketer](https://blog.hubspot.com/), Figma teaches you how to be a [better designer](https://www.figma.com/blog/insights/))
+
+- Share uncomfortable things like [postmortems](/blog/nov-24-shai-hulud-attack-post-mortem) for incidents 
+
+- Be a good citizen, [create space](/handbook/people/hogpatch) for others to thrive, help developers (or whichever community you market to) without asking for anything in return
+
+- Turn internal knowledge into [public artifacts](/handbook) and give space for every role to author content that matters to them
+
+## Fast, not frantic
+
+It's tempting to treat “shipping too fast” as a humblebrag, but that’s lazy thinking. If users can’t adopt what we ship, it’s not velocity – it’s waste.
+
+Practically, that means we need to be explicit about what matters, get better at sequencing the big launches, make a select number of things very loud, and let everything else be quietly excellent. 
+
+And of course, do more things that would make a normal SaaS company nervous. Not everyone will like it, and that’s what makes velocity convert to value.
```

Comment:
Combining the last two paragraphs as I don't fully understand what you mean in the last one. 
```suggestion
Practically, that means we need to be explicit about what matters, doing things that would make normal SaaS companies nervous, get better at sequencing the big launches, make a select number of things very loud, and let everything else be quietly excellent. Doing this right helps velocity convert to value.
```